### PR TITLE
Miscellaneous Fixes for Demo

### PR DESCRIPTION
**Pull request by rawfalafel**
- Processing the first block in a cycle causes an out of bounds error [here](https://github.com/prysmaticlabs/prysm/blob/4c5099c8ba2162270d02f5004bf745c4be7b27bd/beacon-chain/casper/validator.go#L78). There's already a discussion at https://github.com/ethereum/eth2.0-specs/issues/92 to resolve the issue. In the meantime, this changeset adds a third cycle worth of committees so that the chain can proceed.
- The GHOST fork choice rule is modified so that when `crystallized_state_root` values are unequal but `last_finalized_slot` and `last_justified_slot` are equal, `slot` is used as a tiebreaker. This allows the chain to continue even when slots never get finalized.
- I also modified `cycleLength` to 2 for the demo config (@terenc3t 's idea!)

### CODE REVIEW

**Comment by rawfalafel** [comment]
Ah, can you not merge yet? I want to double check if the change I made in
genesis.json is correct.

**Comment by terenc3t** [comment]
Changes in genesis.json looks correct, I had to do the samething in #680

**Comment by rauljordan** [comment]
+1 to why remove some functions @rawfalafel

**Comment by rawfalafel** [comment]
I got rid of `CommitteeInShardAndSlot` because it shared duplicate logic with `GetShardAndCommitteesForSlot`. 

I got rid of `TestQuadraticPenaltyQuotient` because the function it's testing is weird; all it's doing is squaring a constant, and I think there might be a bug in there anyways. I added the test back in for now.

File: beacon-chain/casper/validator.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ func InitialValidators() []*pb.ValidatorRecord {
 	for i := 0; i < params.GetConfig().BootstrappedValidatorsCount; i++ {
 		validators[i] = &pb.ValidatorRecord{
 			Status:            uint64(params.Active),
-			Balance:           uint64(params.GetConfig().DepositSize),
+			Balance:           uint64(params.GetConfig().DepositSize) * uint64(params.GetConfig().Gwei),
```
Comment (rawfalafel) [inline beacon-chain/casper/validator.go]: I converted this to gwei too :smiley:

Comment (terenc3t) [reply beacon-chain/casper/validator.go]: Good call!

**Pull request merged**